File: react/list.js
//#pragma: config 
{
    defaultGetter:"getList",
    defaultAction:"select"
}

//#pragma: execution 
 <div>{##SOURCE##.map(element => <div onClick={() => {let event = element;  ##TRIGGERS##}} key={JSON.stringify(element)}> <b>{ element.title  } </b><small style={{marginBottom:10}}>{ element.subtitle  } </small></div>)}</div>
   

File: flutter/list.dart
//#pragma: setup
listviewchildren[[id]].add(ListTile(title:Text("john")));


//#pragma: execution
##SOURCE##.length > 0 ? Expanded(child:ListView(children:##SOURCE##.indexed.map((v) => ListTile(onTap: () { dynamic event = v.$2;##TRIGGERS## },title:Text('${v.$2.title}'), subtitle:Text('${v.$2.subtitle}'))).toList())) : Expanded(child:Container(alignment:Alignment.center, color:Colors.green, child:Text('no data for list'))),

//#pragma: declaration
    List<ListTile> listviewchildren[[id]]= [];

//#pragma: config
{
    defaultAction:"select",
    defaultGetter:"getList" 
}
